nuxeo-core/nuxeo-core-convert-plugins/src/main/resources/OSGI-INF/commandline-pdf2text-contrib.xml

```diff
@@ -16,6 +16,7 @@
       <winCommand>gswin64c</winCommand>
       <parameterString>-dNOPAUSE -dBATCH -sDEVICE=pdfwrite -dEPSFitPage -sOutputFile=#{targetFilePath}
         #{sourceFilePath}</parameterString>
+      <testParameterString>-dNODISPLAY</testParameterString>
```

Comment:
Why only in tests? I guess we want it also in regular usage...

Reply:
My understanding is that the display happens only when executing the command "gs" without any other parameters: in regular usage, the "parameterString" above prevents the problem.

Reply:
Also, since I seem to be the only annoyed by this issue :D : I don't want to introduce any misbehaviour in the regular usage in a backport.

Reply:
Ah yes sorry I misremembered what `testParameterString` is for, it's for testing if a command is present, not for unit test context...